### DISCUSSION TITLE
Bait Cell: Eyes will see bait cell as food. Harms organisms.

### DISCUSSION THREAD

**Discussion by theomgdev** [MaxRobinsonTheGreat/EvolutionSimulatorV2#73] [Ideas and Suggestions]
#72 
Bait - Blue with a red plus, allows the organism to deceive other organisms. Eyes will see bait as food. Harms organisms in directly adjacent cells (besides itself).

**High lifespan multiplier+bait cell = flesh-eating plants w/o producer cell!**

**Comment by doo158** [comment]
i think a more balanced feature would simply be to remove the killer aspect of the cell, and let organisms evolve a killer cell nearby on their own

**Reply by Coolboy-10** [reply]
Good idea
